vuepress/docs/next/tutorials/devops/external-db.md

```diff
@@ -43,7 +42,7 @@ backup workflow that will scale with your application.
       key: password
       optional: false
 - name: SPRING_JPA_DATABASE_PLATFORM
-  value: org.hibernate.dialect.PostgreSQLDialect
+  value: {ECRDB_DIALECT}
```

Comment:
I compared this section pretty much line by line and made a few changes. Again, put comment tags on lines that should be removed.